### PR TITLE
Advance Javascript session assignment done.

### PR DESCRIPTION
**Pull request by rohitp-webonise**

### CODE REVIEW

**Review by swapnil-webonise — APPROVED** [review]

File: API-Demo/api_Index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            return [];
+        }
+    };
+
+    const displayPosts = async (posts) => {
+        const alternatePosts = posts.filter((_, index) => index % 2 === 0);
+        
+        const postsHtml = alternatePosts.map(({ id, title, body }) => `
+            <div class = "post-Number">Post - ${id} </div>
+            <div class="post" data-post-id="${id}">
```
Comment (mayank-webonise) [inline API-Demo/api_Index.js]: data-post-id="${id}" - is this working? There seems to be some syntax error for id

File: API-Demo/api_Index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                <div class="post-header">
+                    <h2>${title}</h2>
+                    <button class="toggle-comments" data-post-id="${id}">Show Comments</button>
+                </div>
+                <p>${body}</p>
```
Comment (mayank-webonise) [inline API-Demo/api_Index.js:43]: no need to add $, javascript works if you write it inside curly brackets {}.
We use $ while creating a template string that involves variables

**Review by mayank-webonise — COMMENTED** [review]
CHeck comments

File: API-Demo/api_Index.js (hunk 1 above)
Comment (rohitp-webonise) [reply API-Demo/api_Index.js]: Yes its working and I didn't get any error their.

File: API-Demo/api_Index.js (hunk 2 above)
Comment (rohitp-webonise) [reply API-Demo/api_Index.js:43]: Changes done.

**Review by mayank-webonise — APPROVED** [review]

**Pull request merged**